src/pages/mathematics/imaginary-numbers/index.md

```diff
@@ -12,4 +12,4 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
-
+imagenary numbers are not real. They are represented with the number i. i is equivlant to the square root of negative one. You can use i to corectly solve otherwise impossible quadratic equations.
```

Comment:
"Imaginary" numbers are members of the set of complex numbers. They can be represented on a variety of ways, including but not limited to the letter `i`. 
I believe this article should be renamed to "Complex numbers".